Concepts/concept_source.md

```diff
@@ -0,0 +1,106 @@
+# Concept Source Type
+
+## Overview
+
+* A concept maps with the concept source type in openMRS. 
+
+* Concept can have any number of mappings to any number of other vocabularies. 
+
+* Other vocabularies are called "concept sources" in OpenMRS (ie. LOINC, SNOMED, ICD-9, ICD10, RxNORM, etc), but the concept 
+source can also be a custom (ie. org.openmrs.module.mdrtb, PIH, AMPATH, MVP, etc.). 
+
+* Every concept can define a string for its mapping in any "concept source" defined in the database
+
+## Available operations. 
+
+1. [List concept_source types](#list-concept-source-types)
+2. [Create a concept_source](#create-a-concept_source-type)
+3. [Update a concept_source type](#update-a-concept_source-type)
+4. [Delete a concept_source type](#delete-a-concept_source-type)
```

Comment:
Remove the term "type(s)". These are just "concept sources", not "concept source types."